src/planscape/forsys/views.py

```diff
@@ -3,43 +3,321 @@
 import os
 import pandas as pd
 import rpy2
+import sys
 
+from boundary.models import BoundaryDetails
+from conditions.models import BaseCondition, Condition, ConditionRaster
 from django.conf import settings
+from django.contrib.gis.gdal import CoordTransform, SpatialReference, GDALRaster
+from django.contrib.gis.geos import GEOSGeometry, Point, Polygon
 from django.http import (HttpRequest, HttpResponse, HttpResponseBadRequest,
                          JsonResponse, QueryDict)
 
+
 # Configure global logging.
 logger = logging.getLogger(__name__)
 
-# Gets query parameters for scenario_set
-# TODO: clarify input parameters.
-def get_target_value(params: QueryDict):
-  assert isinstance(params['target_value'], str)
-  target_value = int(params['target_value'])
-  return target_value
-
 # Converts R dataframe to Pandas dataframe.
 # TODO: the broadly-accepted solution involves robjects.conversion.rpy2py - debug why it failed with an input type error.
 def convert_rdf_to_pddf(rdf):
   pddf = pd.DataFrame.from_dict({ key : np.asarray(rdf.rx2(key)) for key in rdf.names })
   return pddf
 
+# Converts dictionary of lists to R dataframe.
+# The lists must have equal length.
+def convert_dictionary_of_lists_to_rdf(lists):
```

Comment:
Have you thought about using a pandas Data Frame instead?

Reply:
I did consider passing pandas df instead of a dictionary of numpy lists; however, converting pandas df to R df using robjects.conversion.rpy2py requires more eng investigation/effort and I didn't think it'd be the best use of time when we already have a way to convert a dictionary of numpy lists into an R df.